onnxruntime/core/optimizer/transformer_memcpy.cc

```diff
@@ -109,14 +116,17 @@ This transformer does not currently optimize copies between, e.g., two different
 
 bool TransformerMemcpyImpl::ModifyGraph(const KernelRegistryManager& kernel_registries) {
   bool modified = false;
+  InitializedTensorSet initializers_consumed;
   // find defs that require copy
   for (auto& node : graph_.Nodes()) {
     //don't need to do node placement here now, onnxruntime will do it according to registered kernels.
-    ProcessDefs(node, kernel_registries);
+    //as we process the defs for the nodes in the vurrent graph level, collect the initializers scoped to the current graph level alone
```

Comment:
typo: vurrent->current

Reply:
and per your updated logic, this is not only collecting initializers in current graph level, but also its parent graph.

Reply:
yes that is the intention...

Reply:
I think the point was the comment could be clarified to say that you're collection the initializers consumed at the current graph level. the initializers themselves could come from the current graph or any ancestor, so 'initializers scoped to the current graph level' is potentially confusing.

Reply:
I see. I updated the comment. Thanks.